app/views/users/show.html.erb

```diff
@@ -21,21 +21,23 @@
         </div>
       </div>
       <div class="table-responsive">
-        <table class="table table-bordered table-striped table-hover">
+        <table id="Task-table" class="table table-bordered table-striped table-hover">
           <thead>
             <tr>
               <th>Task Name</th>
               <th>Description</th>
               <th>Date</th>
+              <th>  </th>
             </tr>
           </thead>
 
           <tbody>
             <% @items.each do |item| %>
-              <tr>
+              <tr id=<%= "task-#{item.id}" %>">
```

Comment:
Hi




found the bug in In app/views/users/show.html.erb:


> </tr> > </thead> > > <tbody> > <% @items.each do |item| %> > - <tr> > + <tr id=<%= "task-#{item.id}" %>”>

has an extra “ at the end.  Typo.




changing it to 


> </tr> > </thead> > > <tbody> > <% @items.each do |item| %> > - <tr> > + <tr id=<%= "task-#{item.id}" %>>




I tried various combination.  I can see xhr request and response.  When I copy the response and enter it via chrome dev console, the selector return [].

Now I understand since the extra “ screw up the id which is why it return [].









regardsEG

On Sun, Mar 15, 2015 at 5:26 PM, Eliot Sykes <notifications@github.com>
wrote:

>>              </tr>
>>            </thead>
>>  
>>            <tbody>
>>              <% @items.each do |item| %>
>> -              <tr>
>> +              <tr id=<%= "task-#{item.id}" %>">
> ```html
> <tr id=<%= "task-#{item.id}" %>">
> ```
> The opening double-quote for the id attribute is missing, try:
> ```html
> <tr id="task-<%= item.id %>">
> ```
> That might fix the JS problem mentioned.
> It'd be ok to simplify the selector in the javascript file to just use the id (drop the `tr`): 
> ```html
> $('#task-' + <%= @item.id %>).hide();
> ```
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/yeoenggu/Bloccitoff/pull/4/files#r26445984